torch/_subclasses/meta_utils.py

```diff
@@ -321,6 +325,64 @@ def empty_create(inner_t, inner_src, symbolic_context=symbolic_context):
                         with torch.enable_grad():
                             r = r.clone()
                             r._coalesced_(t.is_coalesced())
+                elif is_sparse_compressed(t):
+                    is_leaf = safe_is_leaf(t)
+
+                    def mk_meta():
```

Comment:
This function is weirdly a lot more complicated than the one in sparse_coo above, not sure if it's needed or not yet

Reply:
I guess the problem is you don't have a constructor that just directly takes sparse/dense dim and allocates those tensor under the hood, do you...

Reply:
Exactly, we don't have `_sparse_coo_tensor_with_dims`-alike for sparse compressed tensors. I could expose one as an aten operator but does it matter if it is either implemented in Python (as above) or in C++? IIRC, their specification will be the same: construct as sparse compressed tensor with indices and values as empty tensors with device='meta' and `nnz==0`.
What do you think?

> ... not sure if it's needed or not yet

It surely appears to be needed as without it a bunch of dynamo tests are failing.